minecraft-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,281 @@
+            "metadata": {
+                "description": "Password for the admin user name."
+            }
+        },
+        "dnsNameForPublicIP": {
+            "type": "string",
+            "metadata": {
+                  "description": "Put a unique DNS Name for the Public IP used to access the Virtual Machine."
+            }
+        },
+        "difficulty": {
+            "type": "string",
+            "defaultValue": "1",
+            "allowedValues": [
+                "0",
+                "1",
+                "2",
+                "3"
+            ],
+            "metadata": {
+                "description": "0 - Peaceful, 1 - Easy, 2 - Normal, 3 - Hard"
+            }
+        },
+        "level-name": {
+            "type": "string",
+            "defaultValue": "world",
+            "metadata": {
+                "description": "Name of your world"
+            }
+        },
+        "gamemode": {
+            "type": "string",
+            "defaultValue": "0",
+            "allowedValues": [
+                "0",
+                "1",
+                "2",
+                "3"
+            ],
+            "metadata": {
+                "description": "0 - Survival, 1 - Creative, 2 - Adventure, 3 - Spectator"
+            }
+        },
+        "white-list": {
+            "type": "string",
+            "defaultValue": "false",
+            "allowedValues": [
+                "false",
+                "true"
+            ],
+            "metadata": {
+                "description": "Only ops and whitelisted players can connect"
+            }
+        },
+        "enable-command-block": {
+            "type": "string",
+            "defaultValue": "false",
+            "allowedValues": [
+                "false",
+                "true"
+            ],
+            "metadata": {
+                "description": "enable command blocks"
+            }
+        },
+        "spawn-monsters": {
+            "type": "string",
+            "defaultValue": "true",
+            "allowedValues": [
+                "false",
+                "true"
+            ],
+            "metadata": {
+                "description": "Enables monster spawning"
+            }
+        },
+        "generate-structures": {
+            "type": "string",
+            "defaultValue": "true",
+            "allowedValues": [
+                "false",
+                "true"
+            ],
+            "metadata": {
+                "description": "Generates villages etc."
+            }
+        },
+        "level-seed": {
+            "type": "string",
+            "defaultValue": " ",
+            "metadata": {
+                "description": "Add a seed for your world"
+            }
+        }
+    },
+    "variables": {
+        "imagePublisher": "Canonical",
+        "imageOffer": "UbuntuServer",
+        "OSDiskName": "osdiskforlinuxsimple",
+        "newStorageAccountName": "[parameters('dnsNameForPublicIP')]",
```

Comment:
Probably make this a parameter. Otherwise somebody could enter myuniquednsname, then it appears as the name of the storage account and the vm